README.rst

```diff
@@ -1353,8 +1353,29 @@ which you don’t care about. The response headers are downloaded always,
 even if they are not part of the output
 
 
+Specifying the raw body
+=======================
+
+Several solutions are available to allow passing the raw body of the request.
+You may choose one of the following methods depending on your use case scenario.
+
+
+Request data from the CLI
+-------------------------
+
+An alternative for situations when ``stdin`` is not convenient to use
+is to specify the raw request body via the ``--request, -r`` argument.
+
+.. code-block:: bash
+
+    $ http --request '{"name": "John"}' pie.dev/post
+
+The data is passed as-is to the request without processing — it is not
+converted into `JSON`_ or `forms`_ data.
```

Comment:
I’d use the following order: `stdin`, `--request`, `@file`.

Here’s my suggestion for the raw request intro as well as for the `--request` section (which should be moved after `stdin`):

```suggestion
Specifying raw request body
============================

In addition to crafting structured `JSON`_ and `forms`_ requests with the `request items`_ syntax, you can provide a raw request body that will be sent without further processing. These two approaches specifying request data (i.e., structured and raw) cannot be combined.

There’re three methods for passing raw request data: piping via ``stdin``, ``--request='data'``, and ``@/file/path``.

Request data via ``--request``
------------------------------

In a situation when piping data via ``stdin`` is not convenient (for example, when generating API docs examples), you can specify the raw request body via the ``--request, -r`` option.

.. code-block:: bash

    $ http --request 'Hello, world!' pie.dev/post

.. code-block:: bash

    $ http --request '{"name": "John"}' pie.dev/post
    
```

Reply:
Way better ^^